Xamarin.Forms.Controls/CoreGalleryPages/PickerCoreGalleryPage.cs

```diff
@@ -23,6 +23,10 @@ protected override void Build(StackLayout stackLayout)
 			var titleContainer = new ViewContainer<Picker>(Test.Picker.Title, new Picker());
 			titleContainer.View.Title = "Title";
 
+			var titleColorContainer = new ViewContainer<Picker>(Test.Picker.Title, new Picker());
```

Comment:
The title is the same as the previous one. Could you add `Color` to title?

Reply:
Done